cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1551,6 +1571,12 @@ pub(super) mod test {
             #   echo "it's gettin' flox in here"
             # '''
 
+            # The `[services]` section of the manifest allows you to define services.
+            # Services defined here use the packages provided by the `[install]` section
+            # and any variables you've defined in the `[vars]` section or `hook.on-activate` script.
+            [services]
+            # postgres.command = "postgres --config-file=pg.conf"
```

Comment:
should we include postgres in the install table?
uncommenting this will likely just not work by default.
The comment touches on requiring packages in the install table, but it would be easier to uncomment two lines than authoring one

Reply:
I'm okay with showing a wide variety of different ideas in the template vs. trying to have one environment that has to show off all of the ideas at once

Reply:
discussed in slack which lead to a broader discussion about simplifying this template. can we do that in a follow-on?